cmd/kubeadm/app/apis/kubeadm/validation/validation_test.go

```diff
@@ -124,12 +124,16 @@ func TestValidateIPFromString(t *testing.T) {
 		ip       string
 		expected bool
 	}{
-		{"", false},           // not valid
-		{"1234", false},       // not valid
-		{"1.2", false},        // not valid
-		{"1.2.3.4/16", false}, // not valid
-		{"1.2.3.4", true},     // valid
-		{"16.0.1.1", true},    // valid
+		{"", false},               // not valid
```

Comment:
Coverage looks great. My main nit is that, when a failure occurs, there is no way to tell which one of the test cases has failed (a problem with the original code). Suggest either displaying the "ip" field, or add a "name" field, which could indicate what is being tested specifically (e.g. no IP, # too large, too short, including mask, ...).